decktape.js

```diff
@@ -102,6 +102,16 @@ parser.script('decktape').options({
     transform : parseRange,
     help      : 'Range of slides to be exported, a combination of slide indexes and ranges (e.g. \'1-3,5,8\')',
   },
+  headed : {
+    default : false,
```

Comment:
I'd suggest to stick to the current default value, which is `true`, and remove the negation when using the option below.